packages/paste-core/components/slider/src/helpers.ts

```diff
@@ -0,0 +1,16 @@
+import type {UIStates, BackgroundColorTokens} from './types';
+
+export const getColorToken = (states: UIStates): BackgroundColorTokens => {
+  if (states.disabled) {
+    return 'colorBackgroundStrong';
+  } else if (states.error) {
+    return 'colorBackgroundError';
+  } else if (states.dragging) {
+    return 'colorBackgroundPrimary';
+  } else if (states.focused) {
+    return 'colorBackgroundPrimaryStronger';
```

Comment:
when the slider is focused and hovered, the color also needs to be colorBackgroundPrimary